### PR TITLE
DOC: added better prose for MDS docs

### PR DESCRIPTION
**Pull request by tacaswell**
Better motivate our design of MDS

### CODE REVIEW

File: source/arch/metadatastore-format.rst [hunk 1]
```diff
@@ -57,31 +57,270 @@ We use the literal ``<time>`` to indicate a client side date/time and
...
+   "seq_num": 1,
+   "data": {"motor1": [2.4, 1431698800.8632061],
+            "photodiode": [0.5, 1431698800.8632062]},
+   "time": 1431698800.8632064
+   "uid": "9912c195-aa5d-4404-b529-823f2a2e30a2"
+  }
+
+For non-scalar data we do not want to store the raw data directly in the events.
+This is both for technical reasons on both the data base side and the collection side
+non-scalar data is stored external to the events and only a reference is stored in
```
Comment (danielballan) [inline source/arch/metadatastore-format.rst:133]: run-on sentence here

File: source/arch/metadatastore-format.rst [hunk 2]
```diff
@@ -57,31 +57,270 @@ We use the literal ``<time>`` to indicate a client side date/time and
...
+   "uid": "9912c195-aa5d-4404-b529-823f2a2e30a2"
+  }
+
+
+We call these documents "Events" and they capture almost all of the
+scientific information about a measurement.  By bundling all of these
+measurements into a single document the experimenter is communicating that
+this set of measurements are "naturally" associated with each other and are,
+to the experimental time scale that matter, "at the same time".  A stream
+of Events represents a single completely synchronous experiment.
```
Comment (danielballan) [inline source/arch/metadatastore-format.rst:159]: I don't think this statement has enough context to mean anything to a novice reader.

File: source/arch/metadatastore-format.rst [hunk 3]
```diff
@@ -57,31 +57,270 @@ We use the literal ``<time>`` to indicate a client side date/time and
...
+  "source": "PV:XF23ID{}-RBV"
+ }
+
+the dtype and shape are book keeping::
+
+
+ {
+  "source": "PV:XF23ID{}-RBV",
+  "dtype": "number",
+  "shape": []
```
Comment (danielballan) [inline source/arch/metadatastore-format.rst:186]: add 'units'

File: source/arch/metadatastore-format.rst [hunk 4]
```diff
@@ -57,31 +57,270 @@ We use the literal ``<time>`` to indicate a client side date/time and
...
+		           "source": "PV:XF23ID{}:motor-RBV",
+			   "dtype": "number",
+			   "shape": []
+			  },
+		"photodiode": {
+		           "source": "PV:XF23ID{}:pd-I",
+			   "dtype": "number",
+			   "shape": []
+			  }
+		  }
```
Comment (danielballan) [inline source/arch/metadatastore-format.rst:221]: again, 'units'

**Comment by danielballan** [comment]
Very exciting to have a nice readable write-up of this! Great!

File: source/arch/metadatastore-format.rst (hunk 3 above)
Comment (tacaswell) [inline source/arch/metadatastore-format.rst:186]: Wary of adding that with almost not discussion.

Comment (danielballan) [inline source/arch/metadatastore-format.rst:186]: fair point

@dchabot @stuwilkins What do you say to adding `units` to the Event Descriptor? This change would not require a migration.

**Comment by danielballan** [comment]
bump @tacaswell you may want to stash this in the metadatastore documentation or perhaps move it into NSLS-II/event-model

closing

**Pull request closed without merging**